src/scripting/level.cpp

```diff
@@ -45,6 +45,21 @@ Level_spawn(const std::string& sector, const std::string& spawnpoint)
   game_session.respawn(sector, spawnpoint);
 }
 
+void
+Level_spawn_transition(const std::string& sector, const std::string& spawnpoint, const std::string& transition)
+{
+  SCRIPT_GUARD_GAMESESSION();
+
+  ScreenFade::FadeType fade_type = ScreenFade::FadeType::NONE;
+
+  if (transition == "fade")
+    fade_type = ScreenFade::FadeType::FADE;
+  else if (transition == "circle")
+    fade_type = ScreenFade::FadeType::CIRCLE;
```

Comment:
An additional `else`, which prints out a warning (using `log_warning`) that the transition provided is not valid, would be great and would allow for consistency with other similar methods (for example, ones that require easing).

```suggestion
  if (transition == "fade")
    fade_type = ScreenFade::FadeType::FADE;
  else if (transition == "circle")
    fade_type = ScreenFade::FadeType::CIRCLE;
  else
    log_warning << "Invalid transition type '" << transition << "'." << std::endl;
```